Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -905,17 +855,11 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
         }
         set
         {
-            var val = newValue
-            if val < 0.0
-            {
-                val = 0.0
+            switch newValue {
```

Comment:
{ another line or?

Reply:
I guess so. Seems weird to me on switch statements.